tests/acceptance/01_vars/01_basic/sys_policy_version.cf

```diff
@@ -0,0 +1,17 @@
+body common control
+{
+  bundlesequence => { "check" };
+  version => "1.2.3";
```

Comment:
Maybe you could also add a test where you don't set this attribute. What should happen then? `sys.policy_version` should be undefined?